icu_benchmarks/models/ml_models/xgboost.py

```
@@ -0,0 +1,86 @@
+import wandb
+from statistics import mean
+# from optuna.integration import XGBoostPruningCallback
+import shap
+
+
+@gin.configurable
+class XGBClassifier(MLWrapper):
+    _supported_run_modes = [RunMode.classification]
+
+    def __init__(self, *args, **kwargs):
+        self.model = self.set_model_args(xgb.XGBClassifier, device="cpu", *args, **kwargs)
```

Ruff

Star-arg unpacking after a keyword argument is strongly discouraged

(B026)